content/docs/deployment/csmoperator/_index.md

```diff
@@ -15,17 +15,23 @@ Dell CSM Operator has been tested and qualified on Upstream Kubernetes and OpenS
 | -------------------------- | ------------------- |
 | 1.25, 1.26, 1.27           | 4.11, 4.12, 4.12 EUS |
 
+>NOTE:
+>- Authorization module is only supported on Kubernetes platforms.
+
+## Supported CSM Components
 
 The table below lists the driver and modules versions installable with the CSM Operator:
 
-| CSM Module         | PowerFlex  |  PowerScale  |  PowerStore |  PowerMax  |  Unity XT  |
-| ------------------ | ---------- | ------------ | ----------- | ---------- | ---------- |
-| CSI Driver         | ✔ 2.5.0+   | ✔ 2.5.0+    | ✔ 2.5.0+    | ✔ 2.6.0+   | ✔ 2.6.0+  |
-| CSM Authorization  | ✔ 1.4.0+   | ✔ 1.4.0+    | ✔ 1.4.0+    | ✔ 1.4.0+   | ✔ 1.4.0+  |
-| CSM Replication    | ✔ 1.3.0+   | ✔ 1.3.0+    | ❌          | ✔ 1.3.0+   | ❌        |
-| CSM Observability  | ✔ 1.3.0+   | ✔ v1.3.0+   | ❌          | ❌         | ❌         |
-| CSM Resiliency     | ✔ 1.6.0+   | ✔ v1.6.0+   |✔ v1.6.0+    | ❌         | ❌         |
+| CSI Driver         | Version | CSM Authorization | CSM Replication | CSM Observability | CSM Resiliency |
+| ------------------ |---------|-------------------|-----------------|-------------------|----------------|
+| CSI PowerScale     | 2.7.0   | 1.6.0             | 1.5.0           | 1.5.0             | 1.6.0          |
+| CSI PowerFlex      | 2.7.0   | 1.6.0             | 1.5.0           | 1.5.0             | 1.6.0          |
+| CSI PowerStore     | 2.7.0   | N/A               | N/A             | N/A               | 1.6.0          |
+| CSI PowerMax       | 2.7.0   | 1.6.0             | 1.5.0           | N/A               | N/A            |
+| CSI Unity XT       | 2.7.0   | N/A               | N/A             | N/A               | N/A            |
 
+>NOTE:
+>- Refer to sample files [here](https://github.com/dell/csm-operator/tree/main/samples) for prior versions of CSM.
```

Comment:
If only 2.7 is supported why giving the samples for older versions ?

Reply:
The note was added in case users wanted to deploy a different version of driver/module other than the latest supported via CSM operator. If you feel this is confusing, we can remove it i suppose.

Reply:
What is confusing is that in the previous table I put the latest supported version picked according to the sample (e.g. 2.5.0+) but you change that value in the table to the latest only. Why ?

Reply:
Because we wanted to retro-actively make the same changes in v2/v3 version as well, so that each of those version will reflect what versions are supported for that release. I thought this might alleviate confusion.  that way we can also capture what versions of modules are supported with that driver release. If we put in, say driver v2.5+ and auth module as 1.3.0+ then it might give an impression that driver v2.6 would work with auth 1.3.0 when we might not have qualified that combination. If we need a call, let us jump on this today and resolve it.